runtime/sema/type.go

```diff
@@ -3259,6 +3259,51 @@ func init() {
 			"Creates a run-time type representing the given static type as a value",
 		),
 	)
+
+	BaseValueActivation.Set(
+		PublicPathType.String(),
+		baseFunctionVariable(
+			PublicPathType.String(),
+			&FunctionType{
+				Parameters: []*Parameter{{
+					Identifier:     "identifier",
```

Comment:
Not specifying an argument label makes the checker assume the identifier to be the argument label. 
Something seems to be wrong here, because the checker test below does not pass the argument label:
https://github.com/onflow/cadence/pull/1211/files#diff-1ddde7bafb81fcdd1cd19e775f812508485a4291b71f533f4f975fe4f9504f95R129

Also, the documentation states there is no argument label required. I'd suggest requiring the argument label `identifier`, to avoid confusion by users which might assume calls like `PublicPath("/public/foo")` work